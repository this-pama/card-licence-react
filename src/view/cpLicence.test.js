import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import CpLicence from './cpLicence';

describe('test cpLicence view page', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CpLicence />, div);
  });

  test('test snapshot of cpLicence view page', () => {
    const component = renderer.create(<CpLicence />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
