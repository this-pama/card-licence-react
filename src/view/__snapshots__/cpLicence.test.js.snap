// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test cpLicence view page test snapshot of cpLicence view page 1`] = `
<div
  style={
    Object {
      "backgroundImage": "logo.svg",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-3 col-sm-2"
      />
      <div
        className="col-md-6 col-sm-8"
        style={
          Object {
            "alignItems": "center",
            "justifyContent": "center",
            "marginTop": 10,
          }
        }
      >
        <div>
          <div
            style={
              Object {
                "backgroundColor": "blue",
                "border": 1,
                "color": "white",
              }
            }
          >
            <h3
              style={
                Object {
                  "padding": 15,
                  "textAlign": "center",
                }
              }
            >
              Register CP Card Licence
            </h3>
          </div>
          <p
            style={
              Object {
                "color": "red",
                "paddingTop": 10,
              }
            }
          />
          <div
            className="form-group"
          >
            <label
              className="form-label"
            >
              Full Name
            </label>
            <input
              className="form-control"
              placeholder="Full Name"
              type="text"
            />
          </div>
          <div
            className="form-group"
          >
            <label
              className="form-label"
            >
              Username
            </label>
            <input
              className="form-control"
              placeholder="Username"
              type="text"
            />
          </div>
          <div
            className="form-group"
          >
            <label
              className="form-label"
            >
              Serial Number
            </label>
            <input
              className="form-control"
              placeholder="Serial Number"
              type="text"
            />
          </div>
          <div
            className="form-group"
          >
            <label
              className="form-label"
              htmlFor="formBasicEmail"
            >
              Email address
            </label>
            <input
              className="form-control"
              id="formBasicEmail"
              placeholder="Enter email"
              type="email"
            />
            <small
              className="text-muted form-text"
            >
              We'll never share your email with anyone else.
            </small>
          </div>
          <button
            className="btn btn-primary btn-block"
            disabled={false}
            type="submit"
          >
               Register CP Licence
          </button>
        </div>
      </div>
      <div
        className="col-md-3 col-sm-2"
      />
    </div>
  </div>
</div>
`;
